dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchFilters.vue

```diff
@@ -176,15 +174,17 @@ export default defineComponent({
      * availableCountries and availableSectors elements in the format expected by the dropdown filters
      */
     async retrieveCountryAndSectorFilterOptions() {
-      const companyDataControllerApi = await new ApiClientProvider(
+      /*const companyDataControllerApi = await new ApiClientProvider(
         assertDefined(this.getKeycloakPromise)(),
-      ).getCompanyDataControllerApi();
+      ).getCompanyDataControllerApi();*/ // TODO Emanuel: Remove if the new way works => Also use in other api calls
```

Comment:
What about this TODO?

Reply:
Removed. All endpoints seem to work due to interceptor in ApiClientProvider.

Reply:
Ok